### PR TITLE
All November updates needs to be merged in Master now.

### PR DESCRIPTION
**Pull request by tarunkethwalia**

### CODE REVIEW

File: src/Components/Start/Start.js [hunk 1]
```diff
@@ -1,39 +1,185 @@
...
+        let userData = userStore.getState().root.user;   
+
+        axios.get(`${env.ApiMonthLink}posts/${userData._id}`)         //POSTS API HIT
+        .then((result)=>{
+            this.setState({
+                ...this.state,
+                posts : result.data.data
+            })
+        },error => {
+            console.log(error);
```
Comment (Sid12261996) [inline src/Components/Start/Start.js]: trigger service for this ... shift any db operation to service level

File: src/Components/Start/Start.js [hunk 2]
```diff
@@ -1,39 +1,185 @@
...
+        }
+
+        // POST CREATE HIT
+        axios.post('https://meetgreet-upload.herokuapp.com/upload',formData,config).then(Imgcreate=>{
+            axios.post(`${env.ApiMonthLink}posts/${userData._id}/create`,
+            {title: this.state.postText, imageUrl : "https://meetgreet-upload.herokuapp.com/images/"+Imgcreate.data}).then(()=>{
+                document.getElementById('postText').value = "";
+                window.location.reload(false);
+            },error=>{console.log(error)});
+        },err=>{console.log(err)});
```
Comment (Sid12261996) [inline src/Components/Start/Start.js]: All these logic has to go in service

File: src/Components/Start/Start.js [hunk 3]
```diff
@@ -1,39 +1,185 @@
...
+                }
+            }
+            else if(this.naturalHeight === this.naturalWidth){
+                document.getElementsByClassName('posty')[index].style.width = "400px";
+                document.getElementsByClassName('posty')[index].style.height = "400px";
+                document.getElementsByClassName('posty')[index].style.filter = "blur(0px)";
+            }
+        });
+        img.src = url;
+    }
```
Comment (Sid12261996) [inline src/Components/Start/Start.js]: add a todo to make this generic utils function... not a priority at this moment but has to be done

File: src/environment.json [hunk 4]
```diff
@@ -1,6 +1,7 @@
 {
   "url": "http://localhost:5000/api/",
   "ApiLink" : "https://meetgreet-api.herokuapp.com/api/",
+  "ApiMonthLink" : "https://meetgreet-api-nov.herokuapp.com/api/",
   "ImageBaseUrl": "https://meetgreet-upload.herokuapp.com/",
   "logo": "https://meetgreet-upload.herokuapp.com/images/1570295217.662374mg02.png"
```
Comment (Sid12261996) [inline src/environment.json]: add todo this has to be cleansed up urgently

Comment (tarunkethwalia) [reply src/environment.json]: JSON standards don't allow comments. Adding this todo in index.js file

**Pull request merged**